public/app/types/dashboard.ts

```diff
@@ -1,5 +1,15 @@
 import { DashboardAcl } from './acl';
 
+export interface Dashboard {
```

Comment:
This type is specific to the search api, can you rename this to DashboardSearchHit and move it to search.ts?